src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -820,6 +820,11 @@ private async Task<IConfidentialClientApplication> BuildConfidentialClientApplic
                     builder.WithRedirectUri(currentUri);
                 }
 
+                // ClientCapabilities are applied once during CCA construction
```

Comment:
I don't know if this comment is needed? Will we have comments like that for each feature?
Maybe we should?

Reply:
I added the comment because a quick code scan suggests WithClientCapabilities() is missing. The comment clarifies that the merge happens earlier, preventing future false alarms.